Deployment/BootMenuNoKVM.ps1

```diff
@@ -1,71 +0,0 @@
-﻿# Copyright (c) Microsoft Corporation. All rights reserved.
-# See LICENSE.txt in the project root for license information. 
-
-<# 
- 
```

Comment:
The new script replaces all functionalities of the current different deployment scripts and files in this folder and also provides additional capabilities. Have different script with the same purpose can be confusing for users.